### PR TITLE
Add Phase-2 KMTF/OMTF/EMTF

### PR DESCRIPTION
**Pull request by zhenbinwu**

### CODE REVIEW

File: python/l1tPh2Nanotables_cff.py [hunk 1]
```diff
@@ -243,6 +243,47 @@
     doc = cms.string("GMT Tk Muons, origin: GMT"),
 )
 # gmtTkMuTable.variables.nStubs = Var("stubs().size()",int,doc="number of stubs")
+### Standalone Muon from GMT, before ghost busting
+
+KMTFpromptMuTable = staMuTable.clone(
+    src = cms.InputTag("l1tKMTFMuonsGmt", "prompt"),
+    name = cms.string("L1KMTFMuon"),
```
Comment (artlbv) [inline python/l1tPh2Nanotables_cff.py:250]: could you put the `XXTF` labels at the end? this will harmonize the naming and make it easier to read

Comment (zhenbinwu) [reply python/l1tPh2Nanotables_cff.py:250]: For the name? Can you show the example here so that I can update it?

Comment (artlbv) [reply python/l1tPh2Nanotables_cff.py:250]: `name = cms.string("L1MuonKMTF"),` would be my proposal

Comment (zhenbinwu) [reply python/l1tPh2Nanotables_cff.py]: Ok, done

**Comment by artlbv** [comment]
it works, though we will need to update the README with proper instructions ...

**Pull request merged**